### PR TITLE
Iconify: add "search all icons" command

### PR DESCRIPTION
**Pull request by Destiner**
## Description

Adds a new command, "Search All Icons", to the Iconify extension. This command runs the query against all icon sets supported by Iconify.

## Screencast

Screenshot 2022-04-09 at 14 13 21

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/iconify/package.json [hunk 1]
```diff
@@ -22,10 +22,17 @@
       "subtitle": "Iconify",
       "description": "View available icons",
       "mode": "view"
+    },
+    {
+      "name": "search-icons",
```
Comment (sxn) [inline extensions/iconify/package.json:27]: would it make sense to just add an `All` item in the `searchBarAccessory` of the other command? 🤔

File: extensions/iconify/src/search-icons.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    setLoading(true);
+    setIcons([]);
+    const icons = await service.queryIcons(text);
+    setIcons(icons);
+    setLoading(false);
+  }
+
+  return (
+    <List throttle isLoading={isLoading} onSearchTextChange={queryIcons}>
+      {icons.map((icon) => {
```
Comment (sxn) [inline extensions/iconify/src/search-icons.tsx:23]: Mind adding an [EmptyView](https://developers.raycast.com/api-reference/user-interface/list#list.emptyview) for when there are no items to show? It's a bit nicer than greeting users with a `No Results` page 😄

File: extensions/iconify/package.json (hunk 1 above)
Comment (Destiner) [reply extensions/iconify/package.json:27]: Hmm. Possible, but dunno. They behave differently both from the implementation and UX perspective.

File: extensions/iconify/src/search-icons.tsx (hunk 2 above)
Comment (Destiner) [reply extensions/iconify/src/search-icons.tsx:23]: 👍 Good one, will add in a bit.

Comment (Destiner) [reply extensions/iconify/src/search-icons.tsx:35]: Added in [41687ff](https://github.com/raycast/extensions/pull/1350/commits/41687ff37950d0771ba70c46b85881f6ef7bf537)

**Comment by sxn** [comment]
🎉

**Pull request merged**